[2024-08-06T21:44:50.164+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: load_parquet_to_bq_dag.dbt_run manual__2024-08-06T21:44:49.418879+00:00 [queued]>
[2024-08-06T21:44:50.172+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: load_parquet_to_bq_dag.dbt_run manual__2024-08-06T21:44:49.418879+00:00 [queued]>
[2024-08-06T21:44:50.172+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-08-06T21:44:50.172+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2024-08-06T21:44:50.172+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-08-06T21:44:50.182+0000] {taskinstance.py:1300} INFO - Executing <Task(DbtRunOperator): dbt_run> on 2024-08-06 21:44:49.418879+00:00
[2024-08-06T21:44:50.185+0000] {standard_task_runner.py:55} INFO - Started process 103 to run task
[2024-08-06T21:44:50.188+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'load_parquet_to_bq_dag', 'dbt_run', 'manual__2024-08-06T21:44:49.418879+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/load_dbt_dag.py', '--cfg-path', '/tmp/tmpds4jhtn8']
[2024-08-06T21:44:50.188+0000] {standard_task_runner.py:83} INFO - Job 61: Subtask dbt_run
[2024-08-06T21:44:50.231+0000] {task_command.py:388} INFO - Running <TaskInstance: load_parquet_to_bq_dag.dbt_run manual__2024-08-06T21:44:49.418879+00:00 [running]> on host cf0df9e834a7
[2024-08-06T21:44:50.261+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=miguel
AIRFLOW_CTX_DAG_ID=load_parquet_to_bq_dag
AIRFLOW_CTX_TASK_ID=dbt_run
AIRFLOW_CTX_EXECUTION_DATE=2024-08-06T21:44:49.418879+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-08-06T21:44:49.418879+00:00
[2024-08-06T21:44:50.262+0000] {dbt_hook.py:117} INFO - dbt run --profiles-dir /opt/dbt_project --models load_parquet_to_test
[2024-08-06T21:44:50.266+0000] {dbt_hook.py:126} INFO - Output:
[2024-08-06T21:44:51.379+0000] {dbt_hook.py:130} INFO - [0m21:44:51  Running with dbt=1.8.4
[2024-08-06T21:44:51.953+0000] {dbt_hook.py:130} INFO - [0m21:44:51  Registered adapter: bigquery=1.8.2
[2024-08-06T21:44:51.963+0000] {dbt_hook.py:130} INFO - [0m21:44:51  Unable to do partial parsing because saved manifest not found. Starting full parse.
[2024-08-06T21:44:52.550+0000] {dbt_hook.py:130} INFO - [0m21:44:52  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2024-08-06T21:44:52.550+0000] {dbt_hook.py:130} INFO - There are 3 unused configuration paths:
[2024-08-06T21:44:52.550+0000] {dbt_hook.py:130} INFO - - models.prazos_project.staging
[2024-08-06T21:44:52.550+0000] {dbt_hook.py:130} INFO - - models.prazos_project.marts.core
[2024-08-06T21:44:52.550+0000] {dbt_hook.py:130} INFO - - models.prazos_project.marts.intermediate
[2024-08-06T21:44:52.618+0000] {dbt_hook.py:130} INFO - [0m21:44:52  Found 1 model, 1 source, 473 macros
[2024-08-06T21:44:52.619+0000] {dbt_hook.py:130} INFO - [0m21:44:52
[2024-08-06T21:44:53.719+0000] {dbt_hook.py:130} INFO - [0m21:44:53
[2024-08-06T21:44:53.721+0000] {dbt_hook.py:130} INFO - [0m21:44:53  Finished running  in 0 hours 0 minutes and 1.10 seconds (1.10s).
[2024-08-06T21:44:53.722+0000] {dbt_hook.py:130} INFO - [0m21:44:53  Encountered an error:
[2024-08-06T21:44:53.723+0000] {dbt_hook.py:130} INFO - Database Error
[2024-08-06T21:44:53.723+0000] {dbt_hook.py:130} INFO -   Invalid project ID 'prazos_project'. Project IDs must contain 6-63 lowercase letters, digits, or dashes. Some project IDs also include domain name separated by a colon. IDs must start with a letter and may not end with a dash.
[2024-08-06T21:44:54.576+0000] {dbt_hook.py:132} INFO - Command exited with return code 2
[2024-08-06T21:44:54.583+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow_dbt/operators/dbt_operator.py", line 98, in execute
    self.create_hook().run_cli('run')
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow_dbt/hooks/dbt_hook.py", line 138, in run_cli
    raise AirflowException("dbt command failed")
airflow.exceptions.AirflowException: dbt command failed
[2024-08-06T21:44:54.585+0000] {taskinstance.py:1318} INFO - Marking task as UP_FOR_RETRY. dag_id=load_parquet_to_bq_dag, task_id=dbt_run, execution_date=20240806T214449, start_date=20240806T214450, end_date=20240806T214454
[2024-08-06T21:44:54.590+0000] {standard_task_runner.py:100} ERROR - Failed to execute job 61 for task dbt_run (dbt command failed; 103)
[2024-08-06T21:44:54.608+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-08-06T21:44:54.615+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
